cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/EquipmentExport.java

```diff
@@ -278,24 +277,72 @@ private static String writeLoadGroups(String networkId, Collection<LoadGroup> fo
     private static void writeLoads(Network network, LoadGroups loadGroups, String cimNamespace, XMLStreamWriter writer, CgmesExportContext context) throws XMLStreamException {
         for (Load load : network.getLoads()) {
             if (context.isExportedEquipment(load)) {
-                LoadDetail loadDetail = load.getExtension(LoadDetail.class);
-                String className = CgmesExportUtil.loadClassName(loadDetail);
+                String className = CgmesExportUtil.loadClassName(load);
```

Comment:
obtainLoadClassName

Reply:
When we decide that we export the EQ profile we determine the "best" match for CGMES load class that corresponds to the current IIDM load (that may have been modified after import). If the call `obtainLoadClassName` here we will be always exporting the original class name.

To be checked with @marqueslanauja

Reply:
Ok.